lib/generators/install/templates/index.coffee.erb

```diff
@@ -0,0 +1,8 @@
+#= require cable
+#= require_self
+#= require_tree .
+
+@<%= application_name %> ?= {}

```

Comment:
Don't think every file should start with that configuration. We should only bend when we have to, so that config should only be output if it's different from the standard config. If not, it should be output in some standard setup.

Not liking `consumer` (singular) being an array either :smile:

Do like @javan's last suggestion :+1: 

Enjoyed the discussion here greatly :heart: